seonoh/ArchitectureStudy/app/src/main/java/com/example/seonoh/seonohapp/TabPagerAdapter.kt

```diff
@@ -10,12 +10,14 @@ class TabPagerAdapter(
 
     private val data = mutableListOf<String>()
 
-    fun setData( data: List<String>){
+    fun setData(data: List<String>) {
         this.data.addAll(data)
```

Comment:
this 에 annotation 을 달아주면, 어떤 인스턴스의 this 인지 알아보기 쉬울 것 같아요.
```suggestion
        this@TabPagerAdapter.data.addAll(data)
```

Reply:
e94a03b63b0c2be405ff607207d9516950787622